augur/distance.py

```diff
@@ -626,7 +626,8 @@ def get_distances_to_all_pairs(tree, sequences_by_node_and_gene, distance_map, e
     return distances_by_node
 
 
-def register_arguments(parser):
+def register_parser(parent_subparsers):
+    parser = parent_subparsers.add_parser("distance", help=first_line(__doc__))
```

Comment:
_(minor, non-blocking)_

Looks like `first_line()` is only useful for the the multi-line docstring in `distance.py` so `augur --help` doesn't overflow. My suggestion:

1. For all but `distance.py`, remove `utils.first_line()` and use `help=__doc__` directly.
2. For `distance.py`, use `help=__doc__.splitlines()[0]`.

Maybe in the future we'll add more info to other docstrings, but it seems a bit overkill currently.

Reply:
Yeah, I agree it's overkill for modules other than `distance.py`. I'll remove `utils.first_line()` from them.

Reply:
Done in 888ba09